modules/server/src/main/ts/bedrock/compiler/Compiler.ts

```diff
@@ -1,13 +1,26 @@
 import * as fs from 'fs';
 import * as Webpack from '../compiler/Webpack';
 import * as Rspack from '../compiler/Rspack';
+import * as Types from './Types';
 
 export interface Compiler {
   readonly generate: () => Promise<Buffer | string>;
 }
 
-export const compile = (bundler: 'webpack' | 'rspack', tsConfigFile: string, scratchDir: string, basedir: string, exitOnCompileError: boolean, files: string[], coverage: string[], polyfills: string[]): Compiler => {
-  const getCompileFunc = () => {
+export const compile = (args: { bundler: Types.Bundler; tsConfigFile: string; scratchDir: string; basedir: string; exitOnCompileError: boolean; files: string[]; coverage: string[]; polyfills: string[];
```

Comment:
There are ways to test development bedrock builds with real development processes, remind me next year and I'd be happy to discuss :+1:

It would be nice to catch these things before we publish a release.